internal/observability/middleware_test.go

```diff
@@ -1531,6 +1652,16 @@ func TestApplicationErrorSnapShot(t *testing.T) {
 						Tags: tags,
 						Unit: time.Millisecond,
 					},
+					{
+						Name: "timeout_ttl_ms",
+						Tags: tags,
+						Unit: time.Millisecond,
+					},
+					{
+						Name: "ttl_ms",
+						Tags: tags,
+						Unit: time.Millisecond,
+					},
```

Comment:
Here we should have a metric point for ttl. Even if it is an application error, we still want to record the ttl. Or do I miss something?

Reply:
ttl histogram wouldn't be recorded as the context does not have any deadline in this test
https://github.com/yarpc/yarpc-go/blob/09b25fb18f441cca055b52d81567b99834e4d974/internal/observability/middleware_test.go#L1586

Reply:
can we update the test and add one?

Reply:
I had added a separate test to check for ttl values. Have updated this test as well